src/Sentry/Sentry.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(SolutionName)' != 'Sentry.Unity'">
-    <TargetFrameworks>net6.0;net5.0;netcoreapp3.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>net8.0;net6.0;net5.0;netcoreapp3.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
     <TargetFrameworks Condition="'$(NO_ANDROID)' == ''">$(TargetFrameworks);net7.0-android</TargetFrameworks>
     <TargetFrameworks Condition="'$(NO_IOS)' == '' And $([MSBuild]::IsOSPlatform('OSX'))">$(TargetFrameworks);net7.0-ios</TargetFrameworks>
     <TargetFrameworks Condition="'$(NO_MACCATALYST)' == '' And $([MSBuild]::IsOSPlatform('OSX'))">$(TargetFrameworks);net7.0-maccatalyst</TargetFrameworks>
```

Comment:
Just out of curiosity, why do we need to target net8.0 specifically? How about we limit this to the AOT solution? something like:

```suggestion
    <TargetFrameworks>net6.0;net5.0;netcoreapp3.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
    <TargetFrameworks Condition="'$(NO_ANDROID)' == ''">$(TargetFrameworks);net7.0-android</TargetFrameworks>
    <TargetFrameworks Condition="'$(NO_IOS)' == '' And $([MSBuild]::IsOSPlatform('OSX'))">$(TargetFrameworks);net7.0-ios</TargetFrameworks>
    <TargetFrameworks Condition="'$(NO_MACCATALYST)' == '' And $([MSBuild]::IsOSPlatform('OSX'))">$(TargetFrameworks);net7.0-maccatalyst</TargetFrameworks>

    <!-- For AOT, we only build a library targeting at least .net 7.0 -->
    <TargetFrameworks Conditon="'$(SolutionName)' == 'Sentry.Aot'">net7.0</TargetFrameworks>
```

Reply:
We could try to make it work for .NET 7 but there's quite a bit of stuff that was added in .NET 8 to help with things like serialization. I'm not sure how much of that can be used in .NET 7 simply by including the new .NET 8 nuget packages as dependencies. For `version 1` of this though, to make things simple, I figured I'd try to get it working with .NET 8 and we could extend that to see if we can make it work for .NET 7 at some point further down the road (once we've also got all the other sentry packages supporting AOT).

Reply:
makes sense